README.md

```diff
@@ -37,26 +50,98 @@ You do not need a RFC for:
 
 If in doubt, just ask on [Gitter](http://gitter.im/inveniosoftware/invenio).
 
-### How to submit a RFC?
+### Step 1: Request an RFC
+
+- [Open an issue](https://github.com/inveniosoftware/rfcs/issues/new/choose).
+  - Document the 1) Motivation 2) Summary of proposed changes and 3) Resources
+
+##### For architects:
+
+- Label and assign the request
+  - All new RFCs should have the "Proposal: Pending" label and a label for the product (e.g. "RDM").
+- Review the request
+- **If rejected**:
+  - Add a justification to the comments of the issue.
+  - Change the label to "Proposal: Rejected".
+- **If accepted**
+  - Change the label to "Proposal: Accepted".
+  - Create a [new CodiMD](https://codimd.web.cern.ch/new) filling it with the [RFC template](0000-template.md)
+  - Add link to CodiMD from issue (``- [RFC work-in-progress](...)``])
+  - Assign the point-of-contact architect
+
+### Step 2: Write the RFC
+
+Here is a couple of advices for collaboratively writing the RFC in an collaborative and efficient manner:
+
+- Choose an editor (person) being responsible for this phase (e.g. the architect or the person opening the RFC request)
+- **Brainstorming phase:**
+  - Fill the template with unstructured bullet points and high-level outline.
+  - Try to clearly define scope - what's included, and what's excluded.
+  - Try to identify multiple options for solutions.
+  - What issues should be addressed?
+  - Identify possible stakeholders, and include them in the discussions.
+- **Reading phase (moderated by editor):**
+  - Add a "Questions" subsection to each section.
+  - Read the RFC and add questions/comments to the questions sections (prefix each question with ``<name>: ...``).
```

Comment:
Add extra information about purpose (collect questions, so that we can get an overview over what issues has to be discussed IRL)